doc/Administration-guide/System-installation/Single-node-installation/index.md

```diff
@@ -266,424 +453,690 @@ Add the MongoDB repository to the /etc/apt/sources.list.d/mongodb.list.
-    $ sudo dpkg -i kaa-node.deb
+   $ sudo dpkg -i kaa-node.deb
    ```
 
-</div><div id="CentOS1" class="tab-pane fade" markdown="1" >
+</div><div id="CentOS_" class="tab-pane fade" markdown="1" >
 
 1. Download the latest RPM package from the [Kaa download page](http://www.kaaproject.org/download-kaa/).
 2. Unpack the downloaded tarball by executing the following command.
 
    ```bash
-    $ tar -xvf kaa-rpm-*.tar.gz
+   $ tar -xvf kaa-rpm-*.tar.gz
    ```
 
 3. Install the Node service by executing the following command.
 
    ```bash
-    $ sudo rpm -i kaa-node.rpm
+   $ sudo rpm -i kaa-node.rpm
    ```
 
-</div></div>
+</div>
+</div>
+</li>
+</ul>
 
 ## Configuration steps
 
 ### SQL database configuration
 
-Check that the PostgreSQL or MariaDB password and username is up to date in the server configuration files.
+You can find SQL database configuration property file templates in "/etc/kaa-node/conf/" folder: mariadb-dao.properties.template file for MariaDB database and postgresql-dao.properties.template file for PostgreSQL.
+
+<ul>
+<li style="list-style-type: none;">
+<ul class="nav nav-tabs">
+  <li class="active"><a data-toggle="tab" href="#maria_conf">MariaDB</a></li>
+  <li><a data-toggle="tab" href="#postgre_conf">PostgreSQL</a></li>
+</ul>
+
+<div class="tab-content">
+<div id="maria_conf" class="tab-pane fade in active" markdown="1" >
+
+Check that the MariaDB login and password is up to date in the server configuration files.
 
 ```bash
+$ cat /etc/kaa-node/conf/admin-dao.properties | grep jdbc_username
+jdbc_username=sqladmin
+
 $ cat /etc/kaa-node/conf/admin-dao.properties | grep jdbc_password
 jdbc_password=admin
-$ cat /etc/kaa-node/conf/dao.properties | grep jdbc_password
+
+$ cat /etc/kaa-node/conf/sql-dao.properties | grep jdbc_username
+jdbc_username=sqladmin
+
+$ cat /etc/kaa-node/conf/sql-dao.properties | grep jdbc_password
 jdbc_password=admin
+```
+In case of the password or username mismatch, edit the configuration file to set a new password.
 
-$ cat /etc/kaa-node/conf/admin-dao.properties | grep jdbc_username
-jdbc_username=admin
-$ cat /etc/kaa-node/conf/dao.properties | grep jdbc_username
-jdbc_username=admin
+```bash
+$ sudo nano /etc/kaa-node/conf/admin-dao.properties
+
+$ sudo nano /etc/kaa-node/conf/sql-dao.properties
+```
+
+</div><div id="postgre_conf" class="tab-pane fade" markdown="1" >
+
+Check that the PostgreSQL password is up to date in the server configuration files.
+
+```bash
+$ cat /etc/kaa-node/conf/admin-dao.properties | grep jdbc_password
+jdbc_password=admin
+
+$ cat /etc/kaa-node/conf/sql-dao.properties | grep jdbc_password
+jdbc_password=admin
 ```
 In case of the password or username mismatch, edit the configuration file to set a new password.
 
 ```bash
 $ sudo nano /etc/kaa-node/conf/admin-dao.properties
-$ sudo nano /etc/kaa-node/conf/dao.properties
+
+$ sudo nano /etc/kaa-node/conf/sql-dao.properties
 ```
 
+</div>
+</div>
+</li>
+</ul>
+
+#### Switching between MariaDB and PostgreSQL
```

Comment:
Please move content of this section to the previous one. You should mention that there are two kinds of SQL DB configurations and MariaDB configuration is used by default. Use only one tabs pane with two tabs holding all DB specific configurations according to DB. I mean that there is no need to mention password configuration in separate section, just provide whole configuration for each DB.